File: PC_C_Char/PC7_Biblioteca_Matematica.cs
class Program
{
    static void Main()
    {
        Console.WriteLine(EsPar(4));
        Console.WriteLine(Factorial(5));
        Console.WriteLine(MaxDe3(8, 3, 10));
        Console.WriteLine(ConvTemp(30));
        Console.WriteLine(AreaRect(5, 4));
    }

    static bool EsPar(int n)
    {
        return n % 2 == 0;
    }

    static int Factorial(int n)
    {
        int resultado = 1;

        for (int i = 1; i <= n; i++)
        {
            resultado = resultado * i;
        }

        return resultado;
    }

    static double MaxDe3(double a, double b, double c)
    {
        double mayor = a;

        if (b > mayor)
        {
            mayor = b;
        }

        if (c > mayor)
        {
            mayor = c;
        }

        return mayor;
    }

    static double ConvTemp(double c)
    {
        return (c * 9 / 5) + 32;
    }

    static double AreaRect(double b, double h)
    {
        return b * h;
    }
}

File: PC_Python/PC7_Biblioteca_Matematica.py
def es_par(n):
    return n % 2 == 0


def factorial(n):

    resultado = 1

    for i in range(1, n + 1):
        resultado = resultado * i

    return resultado


def max_de3(a, b, c):

    mayor = a

    if b > mayor:
        mayor = b

    if c > mayor:
        mayor = c

    return mayor


def conv_temp(c):

    return (c * 9 / 5) + 32


def area_rect(b, h):

    return b * h


print(es_par(4))
print(factorial(5))
print(max_de3(8, 3, 10))
print(conv_temp(30))
print(area_rect(5, 4))